### PR TITLE
Places bought wizard items into your hand instead of on the floor

### PR DESCRIPTION
**Pull request by SteelSlayer**
## What Does This PR Do

Attempts to place bought wizard items like the wand belt, scrying orb, or staves into any open hands you have. If you do not have any open hands, it will then place the item on the floor beneath you.

I also made some of the code more legible by adding proper spacing between text, and adding some line breaks in other procs I came across while making this PR.

## Why It's Good For The Game

Since vendors, chemistry dispensers, syndicate uplinks and so on already have this behavior, I thought it would be nice to have wizard items follow suit. I've gotten quite used to having the items put into my hand and dislike them being put on the floor all of the time.

## Changelog
:cl:
tweak: Bought items from the wizard spell book are first placed into any empty hands. If all hands are full, it places the item on the ground.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -14,11 +14,13 @@
 
 /datum/spellbook_entry/proc/IsSpellAvailable() // For config prefs / gamemode restrictions - these are round applied
 	return 1
-/datum/spellbook_entry/proc/CanBuy(var/mob/living/carbon/human/user,var/obj/item/spellbook/book) // Specific circumstances
+
+/datum/spellbook_entry/proc/CanBuy(var/mob/living/carbon/human/user, var/obj/item/spellbook/book) // Specific circumstances
```
Comment (Fox-McCloud) [inline code/game/gamemodes/wizard/spellbook.dm]: if you're going to clean up spacing, then clean up the implied `var`.

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 2]
```diff
@@ -311,9 +311,12 @@
 			to_chat(src, "<span class='warning'>You are trying to equip this item to an unsupported inventory slot. Report this to a coder!</span>")
 
 /mob/living/carbon/human/put_in_hands(obj/item/W)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/inventory.dm]: `item/I`

File: code/modules/mob/living/carbon/human/inventory.dm [hunk 3]
```diff
@@ -311,9 +311,12 @@
...
-	if(!W)		return 0
-	if(put_in_active_hand(W))			return 1
-	else if(put_in_inactive_hand(W))	return 1
+	if(!W)
+		return 0
+	if(put_in_active_hand(W))
+		return 1
+	else if(put_in_inactive_hand(W))
+		return 1
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/inventory.dm]: `TRUE` and `FALSE`

File: code/game/gamemodes/wizard/spellbook.dm (hunk 1 above)
Comment (SteelSlayer) [reply code/game/gamemodes/wizard/spellbook.dm]: cleaned up all the implied vars I could find in the file

**Comment by TDSSS** [comment]
Neat qol improvement. 👍

**Comment by Fox-McCloud** [comment]
definitely nice; I dunno why it just spawns it on the floor.

**Pull request merged**